types/good-logs/tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+         "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
you do not need this (DOM lib reference)